docs/integrations/destinations/bigquery.md

```diff
@@ -111,15 +111,13 @@ This is the recommended configuration for uploading data to BigQuery. It works b
 * **GCS Bucket Path**
 * **Block Size (MB) for GCS multipart upload**
 * **GCS Bucket Keep files after migration**
-  * See [this](https://cloud.google.com/storage/docs/creating-buckets) for instructions on how to create a GCS bucket.
+  * See [this](https://cloud.google.com/storage/docs/creating-buckets) for instructions on how to create a GCS bucket. The bucket cannot have a retention policy. Set Protection Tools to none or Object versioning.
 * **HMAC Key Access ID**
-  * See [this](https://cloud.google.com/storage/docs/authentication/hmackeys) on how to generate an access key.
-  * We recommend creating an Airbyte-specific user or service account. This user or account will require read and write permissions to objects in the bucket.
+  * See [this](https://cloud.google.com/storage/docs/authentication/managing-hmackeys) on how to generate an access key. For more information on hmac keys please reference the [GCP docs](https://cloud.google.com/storage/docs/authentication/hmackeys)
```

Comment:
the original link didn't go to a place that helped create hmac keys. it just was generic information about the format of hmac keys.

Reply:
Good catch. Would you mind updating the HMAC links in the GCS destination doc as well?

https://docs.airbyte.io/integrations/destinations/gcs

Totally fine if not. I can do that as a follow up.

Reply:
sure. done!